tutorials/tutorial of qa for fast.ipynb

```diff
@@ -0,0 +1,313 @@
+{
+ "cells": [
```

Comment:
in general, this tutorial is not a tutorial.

1. you have no markdown or descriptive text that explains what's happening
2. it's just a bunch of cells with copy-pasted code

Turn this into a real tutorial, please. That means explaining what's happening as you go, and why it's useful to have this QA functionality.

Reply:
Unresolved. You still need to make this tutorial a bit better, I think. See Ross's comments in slack. To change:
- add a markdown box with text briefly explaining what `fast` does, as well as some background context for the tutorial. 
- read through Ross's comments about tutorials in slack and make sure you've done what he said

Your goal should be that somebody who doesn't know what `FAST` is can read your tutorial and leave with an understanding of why the tutorial is important, what's happening, and how it's relevant for the rest of the pipeline.